prac2_login_verification_UI/main.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MaterialApp(
+  home: Loginpage(),
+));
+
+class Loginpage extends StatefulWidget {
+  const Loginpage({Key? key}) : super(key: key);
+
+  @override
+  State<Loginpage> createState() => _LoginpageState();
+}
+
+class _LoginpageState extends State<Loginpage> {
+  bool value=true;

```

Comment:
**Tip**: Whenever you are dealing with instance variables of a class, it is better to follow OOP principles and make them private.
Try to make instance variables private from now on. :sunglasses:

Reply:
Resolved in 8a4acbc.... :+1: